forge/lib/horizon_occlusion_point.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+
+import math
+import forge.lib.cartesian3d as c3d
+import numpy as np
+
+# Constants taken from http://cesiumjs.org/2013/04/25/Horizon-culling/
+rX = 1.0 / 6378137.0
+rY = 1.0 / 6378137.0
+rZ = 1.0 / 6356752.3142451793
```

Comment:
we can avoid duplicating these values, they are already here https://github.com/geoadmin/3d-forge/blob/master/forge/lib/llh_ecef.py